soap/index.js

```diff
@@ -134,7 +181,7 @@ module.exports = yeoman.Base.extend({
 
     var api, i, n, m;
     self.operations = this.operations;
-    self.apis = generator.generateAPICode(this.operations);
+    self.apis = generator.generateAPICode(this.selectedDS.data.name, this.operations); // eslint-disable-line max-len
```

Comment:
Why the eslint override is needed? You can split the call in two lines as follows:
```
self.apis = generator.generateAPICode(this.selectedDS.data.name, 
  this.operations);
```